### ISSUE TITLE
Booting from external drive

### ISSUE THREAD

**Issue by anagno** [hypriot/image-builder-rpi#260]
Hello everyone,

Raspberry pi 3 supports booting from an external drive (https://www.raspberrypi.org/documentation/hardware/raspberrypi/bootmodes/msd.md). When I tried it with the image for hypriot, the operating system would not start:

![img_20180923_224317](https://user-images.githubusercontent.com/5438380/45932810-869eae00-bf82-11e8-913f-f57c20b61a09.jpg)

With a normal pi image, this works without a problem.
I tried also other ways of forcing the boot (https://www.raspberrypi.org/documentation/hardware/raspberrypi/bootmodes/README.md), but I did not manage to make it work. 

Has anyone tried something similar? Is it a bug ?

KR,
anagno

**Comment by anagno** [comment]
After some more digging, I discover that I have to insert an empty sd card in the sd slot of the raspberry. If I do that, then the kernel is booting, but it has a kernel panic

![img_20180924_221904](https://user-images.githubusercontent.com/5438380/45978032-77376780-c04b-11e8-9756-8835d0a20b61.jpg)

**Comment by anagno** [comment]
I found my solution. Everything is working correct and I managed to boot using the hard drive. What I had to do was to change the root parameter in the /boot/cmdline.txt to point to my hard drive (in my case root=/dev/sda2) and afterwards adjust the /etc/fstab to point again to the hard drive. In my case:

proc /proc proc defaults 0 0
/dev/sda1 /boot vfat defaults 0 0
/dev/sda2 / ext4 defaults,noatime 0 1
